Makefile

```diff
@@ -225,6 +225,20 @@ clean:
 protobuf:
 	$(MAKE) -C .. -f cockroach/build/protobuf.mk
 
+# The .go-version-check target is phony so that it is rebuilt every
+# time, but we create a .go-version-check file so that the include
+# directive below only complains about .go-version-check not existing
+# the first time we are run.
+.PHONY: .go-version-check
+.go-version-check:
+	@required=go1.6; \
```

Comment:
Yeah, putting the version in a `.go-version` file is nicer. It also allow `make GOVERS=go1.5` to allow building with an older (or newer) go version for those who know what they are doing.